### PR TITLE
Add prettier, type missing, and split chronobot a bit

### PR DESCRIPTION
**Pull request by Mikaleb**
Add prettier, type missing, and split chronobot a bit

### CODE REVIEW

**Comment by plhery** [comment]
Coool thank you, though please rebase :)

(I already split to 280 in here https://github.com/PLhery/chronodose_twitterbot/commit/209b5dc47bc1389337fb3b62e74efc7b93e0a120)

File: package-lock.json [hunk 1]
```diff
@@ -1,922 +0,0 @@
-{
```
Comment (plhery) [inline package-lock.json:1]: I'm not sure why did you remove this file?

Comment (Mikaleb) [reply package-lock.json:1]: My mistake, I use yarn ;)

**Comment by plhery** [comment]
Thanks a million, yup I think this was necessary as the codebase is slowly growing :)

File: package.json [hunk 2]
```diff
@@ -4,12 +4,19 @@
   "main": "chronobot.js",
   "scripts": {
     "build": "tsc",
-    "start": "node chronobot.js"
+    "start": "node dist/chronobot.js",
+    "login": "node dist/login.js",
```
Comment (plhery) [inline package.json:8]: So cool you moved that here, let's add it to the readme I guess?

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,9 @@
+DEPARTMENTS_TO_CHECK=75,92,93
```
Comment (plhery) [inline .env.example:1]: Let's add a `copy .env.example .env && vim .env` to the readme

File: package.json (hunk 2 above)
Comment (Mikaleb) [reply package.json:8]: Can do, I just did very basic changes for now, I plan to add i18n possibility too since it can come handful for users that live in certain areas that speak English natively (like my wife for example XD ).

File: .env.example (hunk 3 above)
Comment (Mikaleb) [reply .env.example:1]: Done

**Comment by plhery** [comment]
I'm personally not a fan of not using semicolon, as the automatic semi-colon insertion can sometimes be unintuitive and error-prone - with rare edge cases - (similarly to not putting a bracket for a single-line condition/loop) ; but that's a personal opinion and I'm ok keeping it like that for this project :)
image

**Review by plhery — APPROVED** [review]

**Comment by plhery** [comment]
image
I found another reason that I find quite relevant

Let's put back mandatory semicolons as a majority of js devs seem to be used to semicolons

**Comment by Mikaleb** [comment]
> image
> I found another reason that I find quite relevant
> 
> Let's put back mandatory semicolons as a majority of js devs seem to be used to semicolons

No problem

**Review by plhery — APPROVED** [review]

**Comment by plhery** [comment]
I just realised three things:

- I prefer to use kebab-case for files (seems common, I guess one reason is that operating systems do not treat uppercase in filenames the same way) => viteMaDoseApi.ts
- Nitpick: Let's use typescript 3.8's "import type xxx from viteMaDoseApi.ts" (which doesn't import the js file at runtime)
- I'm afraid marker.png won't be found anymore, I'll set __dirname/../marker.png as a path

(I will commit)
